[7.11] Add note dropping support for glibc 2.12 (#93067)
elastic/kibana: docs/migration/migrate_7_11.asciidoc

*via https://github.com/elastic/kibana/pull/80766[#80766]*

[discrete]
[[glibc_212_unsupported]]
=== Dropped support for glibc 2.12

*Details:* {kib} 7.11.0 upgrades the Node.js runtime to version `14.16.0`. This updated runtime requires glibc `2.16`,

> wich drops support for glibc `2.12`-based operating systems.

Reviewer: ```suggestion
which drops support for glibc `2.12`-based operating systems.
```

Author: good catch!

> *Impact*: Supported versions of {kib} will not be impacted. Users will no longer be able to run {kib} on older operating systems wich require glibc `2.12` (example: CentOS 6). Please see our https://www.elastic.co/support/matrix[support matrix] for a list of currently supported operating systems.

Reviewer 2: ```suggestion
*Impact*: Supported versions of {kib} are not impacted. You can no longer run {kib} on older operating systems that require glibc `2.12` (for example, CentOS 6). Refer to our https://www.elastic.co/support/matrix[support matrix] for a list of currently supported operating systems.
```
